lib/course_planner/Factories/factory.ex

```diff
@@ -0,0 +1,56 @@
+defmodule CoursePlanner.Factory do
```

Comment:
what do you think of setting up [ex_machina](https://github.com/thoughtbot/ex_machina) for that?